__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -15,18 +16,114 @@ import {
 import HathorWallet from '../../src/new/wallet';
 import { HATHOR_TOKEN_CONFIG, TOKEN_MELT_MASK, TOKEN_MINT_MASK } from '../../src/constants';
 import transaction from '../../src/transaction';
-import { TOKEN_DATA, WALLET_CONSTANTS } from './configuration/test-constants';
+import { NETWORK_NAME, TOKEN_DATA, WALLET_CONSTANTS } from './configuration/test-constants';
 import wallet from '../../src/wallet';
 import dateFormatter from '../../src/date';
 import { loggers } from './utils/logger.util';
-import { SendTxError } from '../../src/errors';
+import { SendTxError, WalletFromXPubGuard } from '../../src/errors';
 import SendTransaction from '../../src/new/sendTransaction';
 import helpersUtils from '../../src/utils/helpers';
+import walletUtils from '../../src/utils/wallet';
+import { ConnectionState } from '../../src/wallet/types';
+import MemoryStore from '../../src/memory_store'
 
 const fakeTokenUid = '008a19f84f2ae284f19bf3d03386c878ddd15b8b0b604a3a3539aa9d714686e1';
 const sampleNftData = 'ipfs://bafybeiccfclkdtucu6y4yc5cpr6y3yuinr67svmii46v5cfcrkp47ihehy/albums/QXBvbGxvIDEwIE1hZ2F6aW5lIDI3L04=/21716695748_7390815218_o.jpg';
 
 describe('start', () => {
+  it('should reject with invalid inputs', async () => {
```

Comment:
with invalid parameters?

Reply:
Good catch. `inputs` could be confusing in this context.